common/flatpak-transaction.c

```diff
@@ -1213,6 +1215,25 @@ flatpak_transaction_set_no_deploy (FlatpakTransaction *self,
   priv->no_deploy = no_deploy;
 }
 
+/**
+ * flatpak_transaction_set_enable_diskspace_check:
```

Comment:
Might want to consider making it a property of the  `FlatpakTransaction` `GObject` too, so that it can be set at construction time or by language bindings.

Also means you can call `g_object_notify()` on changes to it.